docs/containers/debug-python.md

```diff
@@ -107,7 +107,123 @@ Here is an example of using `dockerServerReadyAction` to launch the browser to o
+
+### For Django Apps
+
+1. In the Dockerfile, comment out the line that adds app code to the container.
+
+    ``` dockerfile
+    #ADD . /app
+    ```
+
+1. Within the `docker-run` task in the `tasks.json` file, create a new dockerRun attribute with a volumes property. This will create a mapping from the current workspace folder (app code) to the `/app` folder in the container.
+
+    ``` json
+    {
+      "type": "docker-run",
+      "label": "docker-run: debug",
+      "dependsOn": [
+        "docker-build"
+      ],
+      "dockerRun": {
+        "volumes": [
+          {
+            "containerPath": "/app", "localPath": "${workspaceFolder}"
+          }
+        ]
+      },
+      ...
+    }
+    ```
+
+1. Edit the python attribute by **removing** `--noreload` and `--nothreading`.
+
+    ``` json
+    {
+      ...
+      "dockerRun": {
+        "volumes": [
+          {
+            "containerPath": "/app", "localPath": "${workspaceFolder}"
+          }
+        ]
+      },
+      "python": {
+        "args": [
+          "runserver",
+          "0.0.0.0:8000",
+        ],
+      }
+      ...
+    }
+    ```
+
+1. Select the **Docker: Python – Django** launch configuration and hit `kb(workbench.action.debug.start)` to build and run your container.
+1. Modify and save an HTML file.
+1. Refresh the browser and validate changes have been made.
+
+### For Flask Apps
+
+1. In the Dockerfile, comment out the line that adds app code to the container.
+
+    ``` dockerfile
+    #ADD . /app
+    ```
+
+1. Within the `docker-run` task in the `tasks.json` file, create a new dockerRun attribute with a volumes property. This will create a mapping from the current workspace folder (app code) to the `/app` folder in the container.
+
+    ``` json
+    {
+      "type": "docker-run",
+      "label": "docker-run: debug",
+      "dependsOn": [
+        "docker-build"
+      ],
+      "dockerRun": {
+        "env": {
+          "FLASK_APP": "path_to/flask_entry_point.py",
+          "FLASK_ENV": "development"
+        },
+        "volumes": [
+          {
+            "containerPath": "/app", "localPath": "${workspaceFolder}"
+          }
+        ]
+      },
+      ...
+    }
+    ```
+
+1. Edit the python attribute by **removing** `--no-reload` and `--no-debugger`.
+
+    ``` json
+    {
+      ...
+      "dockerRun": {
+        "volumes": [
+          {
+            "containerPath": "/app", "localPath": "${workspaceFolder}"
+          }
+        ]
+      },
+      "python": {
+        "args": [
+          "runserver",
+          "0.0.0.0:8000",
+        ],
+      }
+      ...
+    }
+    ```
+
+1. Select the **Docker: Python – Flask** launch configuration and hit `kb(workbench.action.debug.start)` to build and run your container.
+1. Modify and save an HTML file.
```

Comment:
Same idea as above, could be any file type.

Reply:
Thanks again, made that change